frontends/p4/specialize.h

```diff
@@ -127,6 +167,21 @@ class Specialize : public Transform {
     const IR::Node* postorder(IR::Declaration_Instance*) override;
 };
 
+/** @brief Repeats ConstantFolding, TypeChecking, FindSpecializations, and
```

Comment:
This description is too much tied to the implementation.
I would rather say that it specializes all parser and controls that are used in the program.

The post is actually that there are no more parsers and controls with generic types or constructor arguments in the program.